server/odc-migrate/src/main/java/com/oceanbase/odc/migrate/jdbc/common/V42412AddDefaultRiskDetectRules.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.migrate.jdbc.common;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import javax.sql.DataSource;
+
+import org.springframework.jdbc.core.BatchPreparedStatementSetter;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.transaction.support.TransactionTemplate;
+
+import com.oceanbase.odc.common.util.JdbcTemplateUtils;
+import com.oceanbase.odc.core.migrate.JdbcMigratable;
+import com.oceanbase.odc.core.migrate.Migratable;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * @Author: Lebie
+ * @Date: 2024/2/27 15:38
+ * @Description: []
+ */
+@Slf4j
+@Migratable(version = "4.2.4.12", description = "add default risk level detect rules")
+public class V42412AddDefaultRiskDetectRules implements JdbcMigratable {
```

Comment:
for default risk level, we may ristrict rule modification, 
here we may reset rule to emtpy for default risk level

Reply:
Do you mean to add another risk detect rule for the default risk level, besides for the high risk level?

Reply:
Truncate the default risk level detect rules, and do not allow to create default risk level detect rules.